tests/common/Makefile

```diff
@@ -23,19 +23,40 @@ managed:
 	cp $(MANAGED_DLL)* mk/bin/Debug 
 	cp $(MANAGED_DLL)* .
 
+ifeq ($(isWindows),true)
```

Comment:
Instead of this `if` here, I think we can introduce an `EMBEDDINATOR_CMD` here.

In Windows it can be: `$(EMBEDDINATOR_EXE) -p=windows`.
OSX: `mono --debug $(EMBEDDINATOR_EXE) -p=osx`.